napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -400,10 +403,13 @@ def setup_ui(self):
         mov.start()
         self.show_status_btn = QPushButton("Show Status", self)
         self.show_status_btn.setFixedWidth(100)
+        self.search_github_btn = QPushButton("Show Github Plugins", self)
```

Comment:
being a boolean attribute, this feels more like a QCheckBox than a push button no?  You could leave out the `self._search_github` attribute and the `_toggle_search_github` methods entirely, and just have `create_worker(... search_github=self.search_github_checkbox.isChecked() )

Reply:
yeah, I was originally having it as a checkbox, but then I tried to use the push button to make the UI looking more consistent, checkbox looks kinda ugly in the exsiting setup

Reply:
I find it strange not to have a checkbox too, maybe the checkbox can go on the same row as the `Available Plugins` title but aligned to the right hand margin (so just above the blue `install` button) and be fairly discreet
<img width="1192" alt="Screen Shot 2021-02-04 at 7 54 58 PM" src="https://user-images.githubusercontent.com/6531703/106988815-7b98e280-6725-11eb-8a0d-93461710593e.png">